src/components/Mapper/Mapper.jsx

```diff
@@ -44,10 +45,18 @@ function getSelectedLocation(x, y) {
   return location[0];
 }
 
-export default function Mapper() {
+const canvasDimensions = {
+  width: 1244,
```

Comment:
We'll need a better way to assign dimensions later. based on user's screen. Extract to config for now

Reply:
It also means that we would need a way to dynamically assign the coordinates which sounds like a really difficult thing to do.

Reply:
Fair point, not MVP worthy